source/Ship.cpp

```diff
@@ -2323,12 +2323,13 @@ void Ship::DoGeneration()
 	else
 	{
 		// Ramscoops work much better when close to the system center. Even if a
-		// ship has no ramscoop, it can harvest a tiny bit of fuel by flying
-		// close to the star. Carried fighters can't collect fuel or energy this way.
+		// ship has no ramscoop, the vanilla drives have a tiny bit of ramscoop, and thus 
+		// it can harvest a tiny bit of fuel by flying close to the star.
+		// Carried fighters can't collect fuel or energy this way.
```

Comment:
I'm not sure there's any need to mention the vanilla outfits here.

Reply:
I included that mention for clarity so that people recognize that, baring implementation of alternatives or content changes, there is always some ramscoop present. I'm fine with removing that mention if code people deem it not necessary.

Reply:
Given that it is outfit dependent. it is not possible to assume that it is always present. Additionally, if we did remove it from those outfits for whatever reason, even if just to move it ellsewhere, this would be out of date.

Reply:
True. I will remove it, then. But I will wait until it has finished running checks before doing so, so I can fix any errors if they turn up at the same time.